homeassistant/components/egardia/alarm_control_panel.py

```diff
@@ -89,9 +89,7 @@ async def async_added_to_hass(self) -> None:
     @property
     def should_poll(self) -> bool:
         """Poll if no report server is enabled."""
-        if not self._rs_enabled:
-            return True
-        return False
+        return not self._rs_enabled
```

Comment:
Although the transform is correct. The resulting code is more confusing IMO. Previously you at least knew that it was intentional to invert `_rs_enabled`.